spec/fixtures/active_record.rb

```diff
@@ -637,6 +637,10 @@ def self.verify_name_filter(values, _context)
     end
     values
   end
+
+  def self.creatable_fields(context)
+    super(context) + [:id]
```

Comment:
The ID field gets removed from the creatable fields list. This approach is a [recommended way](https://github.com/cerebris/jsonapi-resources/releases/tag/v0.8.3) to get this behavior again.